src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -707,7 +704,7 @@ internal void AppendOutput(IEnumerable<string> output, int outputLength)
                 }
                 else if (insertLineBreak)
                 {
-                    InsertProjectionSpan(lineBreakProjectionSpanIndex, CreateTrackingSpan(_window._outputLineBreakBuffer, _window._lineBreakString));
+                    InsertProjectionSpan(lineBreakProjectionSpanIndex, _window._lineBreakString);
```

Comment:
This just becomes part of the preceding output buffer and has the same content type?